### ISSUE TITLE
Failure when writing to vector file: Path to corresponding vector file is not shown

### ISSUE THREAD

**Issue by tschmetzer** [qgis/QGIS#49567]
### What is the bug or the crash?

I was writing to several output files simultanously and there has been an error:
![image](https://user-images.githubusercontent.com/67278094/182787805-15e85115-91ee-4c4f-876f-f721d7608905.png)
As the message does not show the file it refers to I cannot reconstruct the error source.

### Steps to reproduce the issue

1. Export a vector file to several output sources simultaneously
2. Induce an error on one of the writing processes (probably difficult to do)

### Versions

3.24.3-Tisler

### Supported QGIS version

- [x] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

**Comment by gioman** [comment]
> Export a vector file to several output sources simultaneously

@tschmetzer what is the tool/script you are using?

**Comment by tschmetzer** [comment]
> > Export a vector file to several output sources simultaneously
> 
> @tschmetzer what is the tool/script you are using?

No tool, just a serial native layer export. As the data to be written is large (>10GB) the first, the second and third files end up being written simultaneously.

**Comment by gioman** [comment]
> just a serial native layer export

by "serial" you mean that you started (manually) a new export process before the previous one was ended, right?

**Comment by tschmetzer** [comment]
> > just a serial native layer export
> 
> by "serial" you mean that you started (manually) a new export process before the previous one was ended, right?

Exactly, I started the next export process before the previous one ended. 
By serial I was referring to the starting time meaning I started the second one after starting the first one. By simultaneously I meant that after starting the second one they were running simultaneously until the first one ended.

Maybe this illustration makes it clearer
![image](https://user-images.githubusercontent.com/67278094/183867557-a394f376-6719-4df0-a3f2-687e1390038b.png)

**Comment by gioman** [comment]
> 2\. Induce an error on one of the writing processes (probably difficult to do)

@tschmetzer tried this using a ~1gb ~6M features input and I wasn't able to replicate.

![image](https://user-images.githubusercontent.com/1951107/184211925-9ce98d7b-86c0-4797-9364-f0a438db01d6.png)

In particular as you say i wasn't able to "induce an error in one of the processes", whatever it means.

**Comment by tschmetzer** [comment]
In my initial comment you can read:
> As the message does not show the file it refers to I cannot reconstruct the error source.
This means I coulndn't find out what the error was because I couldn't identify which of the three export files was concerned.

The point of this issue is not to find out what caused the error but to **extend/adapt the error message** so that in the future the user will be able to see the **filename**.

So this filename in red is what is missing for the user to find out what might have been the problem:
![image](https://user-images.githubusercontent.com/67278094/184331190-44717b37-6702-4f1d-8fb9-4ba2d648ea86.png)

**Comment by tschmetzer** [comment]
Is there any information missing? Can we get rid of the feedback mode?

**Comment by tschmetzer** [comment]
@gioman Can we reopen this issue or what is missing to address / understand the problem?

**Comment by nicogodet** [comment]
If I understand correctly it's an UI issue ?
Missing filename/path in error message ?

**Comment by tschmetzer** [comment]
> If I understand correctly it's an UI issue ? Missing filename/path in error message ?

Exactly :)

Also it looks a bit weird that the error message doesn't state anything about the OGR error that actually occurred (OGR error: ) but that's a different issue that can't be handled without knowing the file that caused the error.
